app/src/main/java/com/github/wnder/ExistingPicture.java

```diff
@@ -0,0 +1,271 @@
+package com.github.wnder;
+
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+
+import androidx.annotation.NonNull;
+
+import com.google.android.gms.tasks.OnFailureListener;
+import com.google.android.gms.tasks.OnSuccessListener;
+import com.google.android.gms.tasks.Task;
+import com.google.android.gms.tasks.Tasks;
+import com.google.firebase.firestore.DocumentSnapshot;
+import com.google.android.gms.maps.model.LatLng;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+
+public class ExistingPicture implements Picture{
+    //Storage
+    private Storage storage;
+
+    //Image unique ID
+    private String uniqueId;
+    private Bitmap bmp;
+
+    //Image location
+    private long longitude;
+    private long latitude;
```

Comment:
You store 2 longs here and you wrote a getter that returns a LatLng. Why not store a LatLng right now ?